project1/application.py

```diff
@@ -34,6 +34,7 @@ def register():
     else:
         name = request.form.get("usr")
         user = db.execute("SELECT username FROM users WHERE username = :name", {"name": name}).fetchone()
+        print(user)
         if user is None:
             psw = request.form.get("psw")
             db.execute("INSERT INTO users (username, password, time) VALUES (:name, :psw, :time)", {"name":name, "psw":psw, "time":datetime.datetime.now()})
```

Comment:
write orm queries if possible ie., in python. not SQL code.

Reply:
i will change them today after today's tasks.